### ISSUE TITLE
MacOS 15.0 requires an extra step to run

### ISSUE THREAD

**Issue by gh0sti** [chuchusoft/Sequential#7]
https://www.macrumors.com/2024/08/06/macos-sequoia-gatekeeper-security-change/

To install apps that Gatekeeper blocks, users will need to open up System Settings and go to the Privacy and Security section to "review security information" before being able to run the software.

**Comment by chuchusoft** [comment]
Yes, or you could remove the quarantine xattr which the distribution instructs you to do. From what I gather, Gatekeeper only kicks in if the file is quarantined. If it is not quarantined, Gatekeeper is not run so you would not need to create an exception in the Settings (System Preferences) app. I should mention that I only have 12.7.6. I haven't used the app on 13.x, 14.x or the 15.x betas.

**Comment by gh0sti** [comment]
> Yes, or you could remove the quarantine xattr which the distribution instructs you to do. From what I gather, Gatekeeper only kicks in if the file is quarantined. If it is not quarantined, Gatekeeper is not run so you would not need to create an exception in the Settings (System Preferences) app. I should mention that I only have 12.7.6. I haven't used the app on 13.x, 14.x or the 15.x betas.

I did run that quarantine script and I still had to grant permission via settings app to run. Gatekeeper gets stricter on macOS 15

**Comment by chuchusoft** [comment]
Thanks for the clarification. When macOS 15 comes out, I'll attempt to upgrade to it and see what the simplest way to make Sequential work, and I'll update the Readme with those findings.
